.github/workflows/build.yml

```diff
@@ -169,3 +169,27 @@ jobs:
         with:
           name: e2e-logs
           path: oneclick-logs/
+
+  security_audit:
+    name: Run security audit
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.4
+      - uses: actions-rs/audit-check@v1.2.0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+  cargo-deny:
+    name: Run cargo deny
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        checks:
+          - advisories
+          - bans licenses sources
```

Comment:
Because this project is headed to the CNCF, they have their own license scanning process. So I think we should drop this from here for now, but keep the other ones

Reply:
I am not familiar with the Snyk license scanner, is it as thorough as a Cargo specific one? Does it even have support for Rust?

My experience with FOSSA (the alternative CNCF license scanner) is not that great, so I wouldn't dismiss this one built specifically for Rust.

Reply:
It might not be. I'm fine keeping this around.

Reply:
I completely agree on the FOSSA experience, we gave that a try as well. We had a call with them about our experience and they were quite open about Rust not being a priority for them because they see next to no demand for it. Which is fair enough, I'm not complaining, just giving context :)